google_cloud_compute/plugin.spec.yaml

```diff
@@ -1752,28 +1725,33 @@ actions:
         title: Zone
         description: The name of the zone for this request
         required: true
+        example: us-central1-a
       filter:
         type: string
         title: Filter
         description: Sets a filter expression for filtering listed resources
         required: false
+        example: name = my-firewall-1
       maxResults:
         type: integer
         title: Max Results
         description: The maximum number of results per page that should be returned.
           Acceptable values are 0 to 500. Default 500
         required: false
+        example: 0
       orderBy:
         type: string
         title: OrderBy
         description: Sorts list results by a certain order
         required: false
+        example: creationTimestamp desc
       pageToken:
         type: string
         title: Page Token
         description: Set pageToken to the nextPageToken returned by a previous list
           request to get the next page of results
         required: false
+        example: CkYIpsyE6vCM7QI6OwoCGAEKAiAACgIYAgoHINDkjPrMGgoCGAIKFCoSZGVmYXVsdC1hbGxvdy1pY21wCgoggvCxhNin4rIJ
```

Comment:
Can we remove this input and handle paging automatically for the user in the code?